### DISCUSSION TITLE
Actions in different directory than .github\workflows

### DISCUSSION THREAD

**Discussion by MariuszMielcarski** [github/vscode-github-actions#207] [Q&A]
Hi
I have noticed that the plugin successfully found my workflows in .github\workflows directory - and that works great.
But I have some reusable actions defined in other directory, and it was not identified as GitHub actions by the plugin.
Can configure the plugin somehow to add additional directories to track? To have IntelliSense work for my custom actions too.

**Comment by KetchupOnMyKetchup** [comment]
Hi there,

Reusable workflows specifically are constrained to be in the `.github/workflows` folder:
- "As with other workflow files, you locate reusable workflows in the .github/workflows directory of a repository. Subdirectories of the workflows directory are not supported."
- Reference: https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow

Did you mean composite actions or a different type of action instead by chance? 
- Composite Actions docs: https://docs.github.com/en/actions/creating-actions/creating-a-composite-action

Thanks!
